src/util/triplea/MapXMLCreator/TerritoryDefinitionsPanel.java

```diff
@@ -0,0 +1,157 @@
+package util.triplea.MapXMLCreator;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.FontMetrics;
+import java.awt.Graphics;
+import java.awt.Point;
+import java.awt.event.MouseEvent;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+
+import util.triplea.MapXMLCreator.TerritoryDefinitionDialog.DEFINITION;
+
+/**
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+class TerritoryDefinitionsPanel extends ImageScrollPanePanel
+{
+	
+	private TerritoryDefinitionsPanel()
+	{
+		s_instance = this;

```

Comment:
Variables named "instance" tend to refer to the singleton pattern. It's a bit confusing to then not follow it. If you do want a singelton, pls look at 'singleton as a java enum'.